trendmicro_apex/plugin.spec.yaml

```diff
@@ -41,6 +41,598 @@ connection:
     type: credential_secret_key
     required: true
     example: "CU1874A2-G782-47X1-B6J3-1014A92624BC"
+  skip_entity_ids:
```

Comment:
Let's move these skip variables to the action to which they apply (Execute Agent Actions). This is because we want to keep connections as simple as possible for the user - and in this case these variables only apply to one action.

We should also do this in a way that allows the user to enter any combination of IP, MAC, AgentID, in a single field. We can auto-detect the type using string pattern matching / regex and apply it so the user experience is much better.